moulinette/authenticators/__init__.py

```diff
@@ -31,6 +32,7 @@ class BaseAuthenticator(object):
 
     def __init__(self, name):
         self._name = name
+        self.is_authenticated = False
```

Comment:
What is this supposed to do? It is set to `True` at some point and never switched back to `False`. `is_authenticated` is global (shared for all users)?

Reply:
Cheers, that wasn't so clear for me either xD